src/models/command-type.ts

```diff
@@ -1,10 +1,11 @@
 import Joi from 'joi';
 
 import { schema as baseFieldConfigurationSchema, BaseFieldConfiguration } from './fields/base-field-configuration';
+import { schema as stringOrTranslationsSchema, StringOrTranslations } from './string-or-translations';
 
 const schema = (apiVersion: number): Joi.ObjectSchema => Joi.object().keys({
   hashId: Joi.string().required().example('x18a92'),
-  name: Joi.string().required().example('Measurement cycle'),
+  name: stringOrTranslationsSchema.required().example('Measurement cycle'),
```

Comment:
`name` should be string for apiVersion <= 6 (also applicable to other cases in this PR)